pwiz_tools/Skyline/Controls/Startup/ActionTutorial.cs

```diff
@@ -187,7 +187,13 @@ public void ExtractTutorial(ILongWaitBroker waitBroker)
                 var skyFileToOpen = Path.Combine(extractDir ?? string.Empty, SkyFileLocationInZip);
                 foreach (var entry in zip.Entries.ToList())
                 {
-                    entry.FileName = entry.FileName.Substring(entry.FileName.IndexOf('/')); // Gets rid of everything up to and including first '/'.
+                    // Remove top level of directory structure if any - unless it's a data directory

```

Comment:
Should we just change the .zip file that is on the website instead of having this logic here?

Reply:
Yes. Tutorial ZIP files have a defined structure where they always include a toplevel folder that includes the name of the tutorial (e.g. MethodEdit, MethodRefine, iRT, etc.) That way things don't become a mess if you just right-click > Extract All to the current folder. This tutorial ZIP file SmallMoleculeLibraries.zip should contain a single subfolder "SmallMoleculeLibraries" which contains all the tutorial files and where the user will be instructed to save the .sky file.

Reply:
There are several other tutorial data sets that don't have a top level directory - I suspect that this code was the cause of some exception reports about "an item with this key has already been added", which the zip library may throw in this case.

Reply:
But according to Brendan such data sets are ill-formed, so I'll back this out and update the data set as you suggest. As to the others, I guess they're not causing trouble so I'll leave them alone.